client/extensions/woocommerce/app/order/order-notes/index.js

```diff
@@ -1,34 +1,127 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { localize, moment } from 'i18n-calypso';
 import PropTypes from 'prop-types';
-import { localize } from 'i18n-calypso';
+import React, { Component } from 'react';
+import { sortBy, keys } from 'lodash';
 
 /**
  * Internal dependencies
  */
+import { areOrderNotesLoaded, getOrderNotes } from 'woocommerce/state/sites/orders/notes/selectors';
 import Card from 'components/card';
+import { fetchNotes } from 'woocommerce/state/sites/orders/notes/actions';
+import OrderNote from './note';
+import OrderNotesByDay from './day';
 import SectionHeader from 'components/section-header';
 
+function getSortedNotes( notes ) {
+	const notesByDay = {};
+	notes.forEach( note => {
+		const day = moment( note.date_created_gmt ).format( 'YYYYMMDD' );
+		if ( notesByDay[ day ] ) {
+			notesByDay[ day ].push( note );
+			notesByDay[ day ] = sortBy( notesByDay[ day ], 'date_created' ).reverse();
+		} else {
+			notesByDay[ day ] = [ note ];
+		}
+	} );
+	return notesByDay;
+}
+
 class OrderNotes extends Component {
 	static propTypes = {
-		order: PropTypes.object,
+		orderId: PropTypes.number.isRequired,
+		siteId: PropTypes.number.isRequired,
 	}
 
-	render() {
-		const { order, translate } = this.props;
-		if ( ! order ) {
-			return null;
+	constructor( props ) {
+		super( props );
+		this.state = {
+			openIndex: 0,
+		};
+	}
+
+	componentDidMount() {
+		const { siteId, orderId } = this.props;
+
+		if ( siteId ) {
+			this.props.fetchNotes( siteId, orderId );
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		if ( newProps.orderId !== this.props.orderId || newProps.siteId !== this.props.siteId ) {
+			this.props.fetchNotes( newProps.siteId, newProps.orderId );
+		}
+	}
+
+	toggleOpenDay = ( index ) => {
+		this.setState( () => ( { openIndex: index } ) );
+	}
+
+	renderNotes = () => {
+		const { days, notesByDay, translate } = this.props;
+		if ( ! days.length ) {
+			return (
+				<p>{ translate( 'No activity yet' ) }</p>
```

Comment:
Hi! I've found a possible matching string that has already been translated **83** times:
`translate( 'No activity yet!' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-orders-view-notes)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).